src/components/manage/Display/Display.jsx

```diff
@@ -101,6 +101,12 @@ const customSelectStyles = {
     ':active': {
       backgroundColor: null,
     },
+    span: {
+      flex: '0 0 auto',
+    },
+    svg: {
```

Comment:
@sneridagh Is this actually a CSS property, or a typo?

Reply:
`react-select` uses Emotion for inner styling. In Emotion you can actually do this, create inner selectors from the styling properties.